src/client/datascience/commands/exportCommands.ts

```diff
@@ -69,8 +69,10 @@ export class ExportCommands implements IDisposable {
             : this.notebookProvider.activeEditor;
 
         if (editor) {
+            await workspace.saveAll(false);
+            const contents = await this.fs.readFile(editor.file);
             const interpreter = editor.notebook?.getMatchingInterpreter();
```

Comment:
Why can't we use `generateCellsFromNotebookDocument` or similar.

Reply:
Because we don't have a NotebookDocument to work with in webview interactive window and webview notebooks. Eventually we can do that, but not yet

Reply:
I don't think Peng and I are planning to remove the ability to opt out of native interactive window till next iteration

Reply:
Oh wait, 🤦‍♀️ this shouldn't be here

Reply:
Since we are disallowing users using custom editor, we should probably stop supporting the webview IW as well.